operator/apis/execution/v1/parsedefinition_types.go

```diff
@@ -33,6 +33,8 @@ type ParseDefinitionSpec struct {
 	Image string `json:"image,omitempty"`
 	// ImagePullSecrets used to access private parser images
 	ImagePullSecrets []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
+	// Image pull policy. One of Always, Never, IfNotPresent. Defaults to Always if :latest tag is specified, or IfNotPresent otherwise. Cannot be updated. More info: https://kubernetes.io/docs/concepts/containers/images#updating-images
```

Comment:
```suggestion
	// Image pull policy. One of Always, Never, IfNotPresent. Defaults to Always if :latest tag is specified, or IfNotPresent otherwise. More info: https://kubernetes.io/docs/concepts/containers/images#updating-images
```